src/ui/components/VPNPopupButton.qml

```diff
@@ -35,11 +35,11 @@ VPNButtonBase {
     contentItem: VPNInterLabel {
```

Comment:
Could this also become VPNMetropolisLabel {} ?

Reply:
According to the design change it seems like the _cancel_ and _remove_ buttons have a different font now and the `font.family` property is set dynamically now.